classes/email/settings.php

```diff
@@ -67,8 +67,8 @@ public static function get_settings() {
 			self::$settings = get_option( self::$option_key, array() );
 			self::$settings = wp_parse_args( self::$settings, self::$defaults );
 		}
-
-		return self::$settings;
+		
+		return apply_filters( 'change_email_settings', self::$settings, $arg1, $arg2 );
```

Comment:
@riverBanjo This filter is in the right place. A few concerns:

1. Filter names should be prefeixed with `caldera_forms_`
1. $arg1 and $arg2 are undefined. Please delete those.
1. There are no inline docs 
    - See: https://github.com/CalderaWP/Caldera-Forms/blob/master/contributing/php-coding-standards.md#inline-documentation
     - See: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters